packages/browser/src/browser/index.ts

```diff
@@ -162,11 +164,12 @@ async function flushFinalBuffer(
   analytics: Analytics,
   buffer: PreInitMethodCallBuffer
 ): Promise<void> {
+  const calls = getGlobalAnalytics()
```

Comment:
Can we get rid of this now? restore the old function signature of "popSnippetWindowBuffer"?

Reply:
Yea I guess we don't ned this anymore since we store it globally